Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -31,11 +76,22 @@ void OnSelect()
                 if (placing)
                 {
                     SpatialMappingManager.Instance.DrawVisualMeshes = true;
+
+                    Debug.Log(gameObject.name + " : " + "Removing existing world anchor if any.");
```

Comment:
what does "if any" mean here? There's no code guarding the DestroyImmediate call if there isn't a WorldAnchor on this gameObject. Does DestroyImmediate do its own null-check internally?